src/services/ethereum/CustomSigner.js

```diff
@@ -0,0 +1,104 @@
+// @flow
```

Comment:
where are the tests for this file? It's a critic part of our system.

Reply:
Added test for the sign function